### PR TITLE
Update pl.coffee

### PR DESCRIPTION
**Pull request by fatherJS**
Update pl.coffee

### CODE REVIEW

File: app/locale/pl.coffee [hunk 1]
```diff
@@ -11,13 +11,13 @@ module.exports = nativeDescription: "polski", englishDescription: "Polish", tran
 #    built_for_teachers_subtitle1: "Computer Science"
 #    built_for_teachers_subblurb1: "Starting with our free Introduction to Computer Science course, students master core coding concepts such as while/for loops, functions, and algorithms."
     built_for_teachers_subtitle2: "Tworzenie gier"
-#    built_for_teachers_subblurb2: "Learners construct mazes and use basic input handling to code their own games that can be shared with friends and family."
+#    built_for_teachers_subblurb2: "Studenci tworzą labirynty i używają klawiatury, aby zaprogramować własne gry i dzielić się nimi ze znajomymi i rodziną."
     built_for_teachers_subtitle3: "Tworzenie stron internetowych"
     built_for_teachers_subblurb3: "Używając HTML, CSS i jQuery, uczniowie wykazują się kreatywnością abt stworzyć ich własne strony internetowe z ustalonym adresem URL, który mogą udostępnić go swoim kolegom/koleżankom z klasy."
-#    century_skills_title: "21st Century Skills"
+#    century_skills_title: "Kompetencje 21-ego wieku"
```
Comment (harnash) [inline app/locale/pl.coffee:17]: How about:
```suggestion
#    century_skills_title: "Umiejętności 21-ego wieku"
```

**Review by harnash — CHANGES_REQUESTED** [review]

**Comment by Bryukh** [comment]
Hi @fatherJS. Thanks for the translations! So I can merge this in, can you [sign the CLA](https://contributors.codecombat.com/cla)?

File: app/locale/pl.coffee (hunk 1 above)
Comment (mloskot) [reply app/locale/pl.coffee:17]: If would be nice if you @fatherJS could apply the suggestion above from  @harnash
I also think the `Umiejętności` fits the context better.

**Comment by Bryukh** [comment]
Closed as CLA not signed. Feel free to reopen it.

**Comment by mloskot** [comment]
@Bryukh I have re-submitted this PR as https://github.com/codecombat/codecombat/pull/6210 and I'm willing sign CLA

**Pull request closed without merging**